substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/hub/DynamicHub.java

```diff
@@ -1105,6 +1118,10 @@ private String computePackageName() {
         return pn;
     }
 
+    private String computeSimpleName() {
```

Comment:
Where is this used?

Reply:
Not. That was a left-over, I'll remove it.